### PR TITLE
Add support for service ATG

### LINKED ISSUE #458: Feature request - ATGplay
Sample url: http://www.atgplay.se/video/20160923-vinnarev64del2
Base API url: www.atgplay.se/api
link to api based on sample url:  www.atgplay.se/api/1474721149001/video/20160923-vinnarev64del2

Looks like they're using RTMP, MPEG4, HLS, RTSP and BAMB.

### PR DESCRIPTION
**Pull request by dalgr**
fixes #458

### CODE REVIEW

File: lib/svtplay_dl/service/atg.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+        try:
+            janson = json.loads(video_assets.text)
+        except json.decoder.JSONDecodeError:
+            yield ServiceError("Can't decode api request: {0}".format(video_assets.text))
+            return
+
+        if "title" in janson:
+            directory = os.path.dirname(self.options.output)
+            title = re.sub(r"[^\S\n\t]+|\\|/", "_", janson["title"])
```
Comment (spaam) [inline lib/svtplay_dl/service/atg.py]: just curious do utils.filenamify generate a bad filename?

**Comment by dalgr** [comment]
@spaam 

I tested with:
```
svtplay-dl https://www.atgplay.se/video/20180127-v75klubbeninforkalmar271 -v
```
And got file name: `atg.play.ts`
With use of tiltel i get: `V75-klubben_inför_Kalmar_27_1.ts`

**Comment by spaam** [comment]
i was thinking `utils.filenamify(janson["title"])`

**Comment by dalgr** [comment]
@spaam 
ok, maby I looked to fast :)
I have updated to us `utils.filenamify(janson["title"])` and it seems to work great.

File: lib/svtplay_dl/service/atg.py (hunk 1 above)
Comment (dalgr) [reply lib/svtplay_dl/service/atg.py]: I have updated to use utils.filenamify(janson["title"])

**Pull request merged**